api/v1beta1/grafanalibrarypanel_types.go

```diff
@@ -0,0 +1,124 @@
+package v1beta1
+
+import (
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// GrafanaLibraryPanelSpec defines the desired state of GrafanaLibraryPanel
+// +kubebuilder:validation:XValidation:rule="(has(self.folderUID) && !(has(self.folderRef))) || (has(self.folderRef) && !(has(self.folderUID))) || !(has(self.folderRef) && (has(self.folderUID)))", message="Only one of folderUID or folderRef can be declared at the same time"
+// +kubebuilder:validation:XValidation:rule="((!has(oldSelf.uid) && !has(self.uid)) || (has(oldSelf.uid) && has(self.uid)))", message="spec.uid is immutable"
+type GrafanaLibraryPanelSpec struct {
+	GrafanaCommonSpec  `json:",inline"`
+	GrafanaContentSpec `json:",inline"`
+
+	// UID of the target folder for this dashboard
+	// +optional
+	FolderUID string `json:"folderUID,omitempty"`
+
+	// Name of a `GrafanaFolder` resource in the same namespace
+	// +optional
+	FolderRef string `json:"folderRef,omitempty"`
+
+	// plugins
+	// +optional
+	Plugins PluginList `json:"plugins,omitempty"`
+}
+
+// GrafanaLibraryPanelStatus defines the observed state of GrafanaLibraryPanel
+type GrafanaLibraryPanelStatus struct {
+	GrafanaCommonStatus  `json:",inline"`
+	GrafanaContentStatus `json:",inline"`
+}
+
+//+kubebuilder:object:root=true
+//+kubebuilder:subresource:status
+
+// GrafanaLibraryPanel is the Schema for the grafanalibrarypanels API
+// +kubebuilder:printcolumn:name="No matching instances",type="boolean",JSONPath=".status.NoMatchingInstances",description=""
+// +kubebuilder:printcolumn:name="Last resync",type="date",format="date-time",JSONPath=".status.lastResync",description=""
+// +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp",description=""
```

Comment:
I missed this yesterday 😅 
```suggestion
// +kubebuilder:printcolumn:name="Last resync",type="date",format="date-time",JSONPath=".status.lastResync",description=""
// +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp",description=""
// +kubebuilder:resource:categories={grafana-operator}
```